docs/docs_skeleton/docs/modules/memory/chat_messages/index.mdx

```diff
@@ -8,7 +8,7 @@ Head to [Integrations](/docs/integrations/memory/) for documentation on built-in
 :::
 
 One of the core utility classes underpinning most (if not all) memory modules is the `ChatMessageHistory` class.
-This is a super lightweight wrapper which provides convenience methods for saving HumanMessages, AIMessages, and then fetching them all.
+This is a super lightweight wrapper that provides convenient methods for saving HumanMessages, AIMessages, and then fetching them all.
```

Comment:
```suggestion
This is a super lightweight wrapper that provides convenience methods for saving HumanMessages, AIMessages, and then fetching them all.
```

Reply:
@HTahboub, Understood, I reverted that changes.

Thanks